images/base/Dockerfile

```diff
@@ -112,6 +112,7 @@ RUN echo "Installing Packages ..." \
       libseccomp2 pigz \
       bash ca-certificates curl rsync \
       nfs-common fuse-overlayfs \
+      jq \
```

Comment:
We should not leave jq in the image only to use it at build time.

Reply:
Done